po/pl.po

```diff
@@ -1953,20 +1951,18 @@ msgid "Internal parse error"
 msgstr "Wewnetrzny błąd parsera"
 
 #: rust/libnewsboat/src/fslock.rs:57
-#, fuzzy
 msgid "Failed to open lock file '%s': %s"
-msgstr "Nie udało się otworzyć pliku blokady: '%s' (%s)"
+msgstr "Nie udało się otworzyć pliku blokady: '%s': %s"
```

Comment:
There are one too many colons here, is this intended? The first `%s` is a filename, the second one is a detailed error message.